### DISCUSSION TITLE
Reolink Go Plus battery powered 4G camera, will it work with neolink?

### DISCUSSION THREAD

**Discussion by LarsKaptein** [thirtythreeforty/neolink#285] [Q&A]
Hi,

I am currently working on a project that requires construction site camera's.  The Reolink Go Plus battery powered 4G cam would be ideal for this, as it can also be charged via a solar panel which reolink offers. But only having access to it via the reolink app/client is a dealbreaker. 

Should this camera work with neolink? Has anybody tested it?

I am ordering one as we speak, so if nobody tested this camera before, i will.

**Comment by vlebourl** [comment]
I don't know about that particular camera, but I can confirm that it's working with both the Argus 2 battery powered cam and the Argus PT. I do recommend plugging them to a power socket however as the stream drains the battery very rapidely, even when powered by a solar panel...

**Reply by LarsKaptein** [reply]
Thanks for the tip!
Do you know if it stops streaming when there are no active rtsp streams? They only want to occasionally check the stream, it won't be recorded or watched 24/7.

**Reply by vlebourl** [reply]
Currently the stream is pulled constantly. You can follow this issue regarding that question: https://github.com/thirtythreeforty/neolink/issues/204

**Comment by PrplHaz4** [comment]
I'm in a very similar situation - with the addition that I'm looking to utilize snapshots more than direct RTSP streaming.  I haven't gotten much further than pulling wireshark traces at this point.

There is also this fork of a different project that looks to be solving specifically for battery/4G cameras which may be helpful.
https://github.com/vherrlein/camera_proxy/tree/develop
